dataclients/kubernetes/testdata/ingressV1/ingress-data/wildcard-ing-prefix.eskip

```diff
@@ -0,0 +1,3 @@
+kube_foo__qux____example_org_____qux:
+	Host("^([a-z0-9]+(-[a-z0-9]+)?[.]example[.]org[.]?(:[0-9]+)?)$") && PathSubtree("/")
```

Comment:
I think you want 
`^([a-z0-9][a-z0-9-]*)?[.]example....`

Better also to write down in the issue what kind of match or regexp you want to create.

Reply:
I want to match a valid hostname, it can't end or start with `-` I think this `([a-z0-9][a-z0-9-]*)` can end with `-`

Reply:
What about foo-bar-qux.example , that your regexp doesn't match?

Reply:
updated the PR to match it, the new regex `[a-z0-9]+((-[a-z0-9]+)?)*`